lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -850,32 +875,74 @@ sub ansible_show_status {
 
     $args{sap_sid} //= get_required_var('SAP_SID');
     my %common_args = (sdaf_config_root_dir => $args{sdaf_config_root_dir}, sap_sid => $args{sap_sid});
+    my $host_group = '';
 
```

Comment:
I'd like to propose a different pattern to keep data collection separated by the rendering

```
my @reports;

push @reports, {title => 'OS info', text => ansible_execute_command(command => 'cat /etc/os-release', host_group => 'all', %common_args)};

push @reports, {title => 'DB processes', text => ansible_execute_command(command => 'ps -ef | grep hdb', host_group => "$host_group", %common_args)};

...

foreach ( @rteports ) {
    record_info($_{title}, $_{text});
} 
```

Reply:
Yes, it is better.

Reply:
Done.